pct.sh

```diff
@@ -37,23 +37,47 @@ then
+then
+    # TODO https://github.com/jenkinsci/jenkins/pull/4120 problems with workflow-cps → jquery-detached:
+    rm -fv pct-work/structs-plugin/plugin/target/surefire-reports/TEST-InjectedTest.xml
+    rm -fv pct-work/apache-httpcomponents-client-4-api/target/surefire-reports/TEST-InjectedTest.xml
+    rm -fv pct-work/ssh-slaves/target/surefire-reports/TEST-InjectedTest.xml
+    rm -fv pct-work/plain-credentials/target/surefire-reports/TEST-InjectedTest.xml
+    # TODO https://github.com/jenkinsci/jenkins/pull/4099
+    rm -fv pct-work/command-launcher/target/surefire-reports/TEST-hudson.slaves.CommandLauncher2Test.xml
+
+    # TODO wrong detached plugin is being picked up
+    # JavaScript GUI Lib: jQuery bundles (jQuery and jQuery UI) plugin v1.2 is older than required. To fix, install v1.2.1 or later.
+    # we have 1.12.4-1 managed currently
+    rm -fv pct-work/trilead-api/target/surefire-reports/TEST-InjectedTest.xml
+fi
 
 # TODO wrong detached plugin is being picked up
 # Structs Plugin version 1.7 is older than required. To fix, install version 1.20 or later.
 # we have 1.20 managed currently
 rm -fv pct-work/cloudbees-folder/target/surefire-reports/TEST-InjectedTest.xml
 
-# TODO https://github.com/jenkinsci/jenkins/pull/4120 problems with workflow-cps → jquery-detached:
-rm -fv pct-work/structs-plugin/plugin/target/surefire-reports/TEST-InjectedTest.xml
-rm -fv pct-work/apache-httpcomponents-client-4-api/target/surefire-reports/TEST-InjectedTest.xml
-rm -fv pct-work/ssh-slaves/target/surefire-reports/TEST-InjectedTest.xml
-rm -fv pct-work/plain-credentials/target/surefire-reports/TEST-InjectedTest.xml
+# TODO flakey tests related to workflow-job saying it's finished but it still hasn't finished updating the log
+# ref: https://github.com/jenkinsci/workflow-job-plugin/pull/131/files#r291657569
+# https://github.com/jenkinsci/workflow-support-plugin/pull/105
+# https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/130
+# https://github.com/jenkinsci/workflow-basic-steps-plugin/pull/110
+rm -fv pct-work/workflow-basic-steps/target/surefire-reports/TEST-org.jenkinsci.plugins.workflow.steps.CatchErrorStepTest.xml
+rm -fv pct-work/workflow-basic-steps/target/surefire-reports/TEST-org.jenkinsci.plugins.workflow.steps.WaitForConditionStepTest.xml
+rm -fv pct-work/workflow-basic-steps/target/surefire-reports/TEST-org.jenkinsci.plugins.workflow.support.steps.stash.StashTest.xml
+rm -fv pct-work/workflow-support/target/surefire-reports/TEST-org.jenkinsci.plugins.workflow.support.pickles.serialization.SerializationSecurityTest.xml
+rm -fv pct-work/workflow-durable-task-step/target/surefire-reports/TEST-org.jenkinsci.plugins.workflow.steps.durable_task.ShellStepTest.xml
+rm -fv pct-work/workflow-durable-task-step/target/surefire-reports/TEST-org.jenkinsci.plugins.workflow.support.steps.ExecutorStepTest.xml
+rm -fv pct-work/workflow-cps/target/surefire-reports/TEST-org.jenkinsci.plugins.workflow.cps.FlowDurabilityTest.xml
```

Comment:
@timja I do see flakes here but did you try to track this down?

Reply:
```
FlowDurabilityTest.testCompleteAndLoadBuilds expected:<...ne] End of Pipeline
[]> but was:<...ne] End of Pipeline
[Finished: SUCCESS
]>
```

specifically

Reply:
Not sure it’s been a long time since I looked at these flakes